src/index.js

```diff
@@ -0,0 +1 @@
+import './css/style.css';
```

Comment:
**Kindly add plain `HTML` using `ES6`:**
_Use ES6 modules to write modular JavaScript with proper ES6 syntax._
- Usage of `arrow functions` instead of the `function` keyword.
- Usage of `import` and `export` instead of `require` or `module.exports`.
- Proper usage of `const` and `let` keywords.